aiida/common/setup.py

```diff
@@ -1080,7 +1080,8 @@ def set_property(name, value):
     elif type_string == "int":
         actual_value = int(value)
     elif type_string == 'list_of_str':
-        actual_value = value.replace(',', ' ').split()
+        # I expect the results as a list of strings
+        actual_value = value.replace(' ').split()
```

Comment:
This is wrong, replace() should get two parameters? Probably you want only `value.split()`?